src/Symfony/Component/HttpKernel/HttpKernelInterface.php

```diff
@@ -36,7 +36,8 @@ interface HttpKernelInterface
      *
      * @return Response A Response instance
      *
-     * @throws \Exception When an Exception occurs during processing
+     * @throws \Exception When an exception / error occurs during processing
+     * @throws \Throwable When an exception / error occurs during processing in Symfony 5.0
```

Comment:
The method can also throw a throwable even now. The difference is only, that in symfony <5, throwable were not caught but instead terminate instantly. So the annotation can simple be `@throws \Throwable When an exception / error occurs during processing`. Nothing that is relevant for this method changes.

Reply:
In this case it needs to be documented on lower branches as well as they are compatible with PHP 7.

Reply:
Actually I'm pretty sure those `@throws` block like in https://github.com/symfony/symfony/pull/33065#discussion_r340498218 document rethrows so it should be `\Exception` here as long as we only rethrow `\Exception`.

My source is Nicolas comment on https://github.com/symfony/symfony/pull/22411.

Reply:
as nicolas said, we don't document rethrows. so either it's Throwable or we don't need an annotation at all.

Reply:
What I meant was that Nicolas comment confirm that this kind of `@throws` document "rethrows". And that consequently, the rethrow we make in the method should be consistent with the annotation. Whatever, I feel like we are wasting time with this, let's just remove those annotations?!